tee-worker/core-primitives/types/src/extrinsics.rs

```diff
@@ -59,13 +64,13 @@ impl Encode for OpaqueExtrinsicWithStatus {
 
 #[derive(Clone, Eq, PartialEq, Debug)]
 pub struct ParentchainUncheckedExtrinsicWithStatus<Call> {
-	pub xt: UncheckedExtrinsicV4<Call, SubstrateDefaultSignedExtra<PlainTip>>,
+	pub xt: UncheckedExtrinsicV4<Address, Call, PairSignature, ParentchainSignedExtra>,
```

Comment:
I don't think we need `ParentchainUncheckedExtrinsicWithStatus` anymore (or even the whole file) - so we need to basically revert https://github.com/litentry/litentry-parachain/issues/1193

Please do it in a follow-up PR if not solved in this one

Reply:
removed.

Reply:
Can you please double-check the old PR was completely reverted?

I still see left-overs like in `core-primitives/types/Cargo.toml`